### PR TITLE
add test case form dirty check back in

### PR DESCRIPTION
**Pull request by nmorasb**
## MADiE PR

Jira Ticket: [MAT-4664](https://jira.cms.gov/browse/MAT-4664)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/createTestCase/CreateTestCase.test.tsx [hunk 1]
```diff
@@ -86,6 +86,15 @@ jest.mock("@madie/madie-util", () => ({
     getAccessToken: () => "test.jwt",
     getUserName: () => MEASURE_CREATEDBY,
   }),
+  routeHandlerStore: {
+    subscribe: (set) => {
```
Comment (jkotanchik-SB) [inline src/components/createTestCase/CreateTestCase.test.tsx:90]: What's this little line do?

File: src/components/createTestCase/CreateTestCase.test.tsx [hunk 2]
```diff
@@ -86,6 +86,15 @@ jest.mock("@madie/madie-util", () => ({
...
+      // set(measure)
```
Comment (jkotanchik-SB) [inline src/components/createTestCase/CreateTestCase.test.tsx]: What does this line do? Should it be commented out?

File: src/components/routes/TestCaseRoutes.test.tsx [hunk 3]
```diff
@@ -58,6 +58,15 @@ jest.mock("@madie/madie-util", () => ({
     getAccessToken: () => "test.jwt",
     getUserName: () => MEASURE_CREATEDBY,
   }),
+  routeHandlerStore: {
+    subscribe: (set) => {
+      // set(measure)
```
Comment (jkotanchik-SB) [inline src/components/routes/TestCaseRoutes.test.tsx]: Same question. Is this line needed?

**Review by jkotanchik-SB — DISMISSED** [review]

File: src/components/createTestCase/CreateTestCase.test.tsx (hunk 2 above)
Comment (nmorasb) [reply src/components/createTestCase/CreateTestCase.test.tsx]: removed, was brought in from copy/paste

File: src/components/routes/TestCaseRoutes.test.tsx (hunk 3 above)
Comment (nmorasb) [reply src/components/routes/TestCaseRoutes.test.tsx]: removed, was brought in from copy/paste

**Review by sb-prateekkeerthi — APPROVED** [review]

**Review by jkotanchik-SB — APPROVED** [review]

**Pull request merged**